doc/requirements/sequential_cutoff

```diff
@@ -0,0 +1,234 @@
+---
+group: Sequential cutoff
+---
+
+Sequential cutoff feature allows to detect sequential data streams
+and redirect them to backend volume (bypassing the cache).
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting sequential cutoff policy for core
```

Comment:
Please use "cache volume" or "exported object" everywhere instead of "core".
Justification: https://open-cas.github.io/contributing.html#naming-conventions-in-the-documentation

Reply:
The problem is that this is neither. The requirement it about properties of core abstraction, which is not the same as block device instance in Linux.